TWLight/emails/views.py

```diff
@@ -41,17 +42,26 @@ def form_valid(self, form):
         # Adding an extra check to ensure the user is a wikipedia editor.
         try:
             assert self.request.user.editor
+            
             email = form.cleaned_data['email']
-            message = form.cleaned_data['message']
-            ContactUs.new_email.send(
-                sender=self.__class__,
-                user_email=email,
-                editor_wp_username=self.request.user.editor.wp_username,
-                body=message
-            )
-            messages.add_message(self.request, messages.SUCCESS,
-            # Translators: Shown to users when they successfully submit a new message using the contact us form.
-            _('Your message has been sent. We\'ll get back to you soon!'))
+            # Additional validation on the email

```

Comment:
Two thoughts:
1. This should use django's builtin email validator
    https://docs.djangoproject.com/en/1.11/ref/validators/#emailvalidator
2. It should probably happen on the profile page, since that's the place we're actually accepting this input.

Reply:
I've [moved the validation over to users app and made use of the built-in validator](https://github.com/WikipediaLibrary/TWLight/pull/211/commits/24078aec926d0c1baa175270fca8259005e120e8). Couple of question though: 

1. the code validates the email field data on `email_change` form submit. Is that how you pictured it happening? If not, I have no clue as to when or how to validate email from the user's profile page. 
2. the email field of the said form and the email field in `models.py` both happen to be EmailFields and use Django's built-in validators. Assuming what I did was right, why do we need this additional validation on top of that?

Reply:
Yeah, I didn't actually go back through and check the model. Consider my feedback to be declarative, not imperative "ensure we have email validation" rather than "implement email validation". That was my bad. Let's not double-implement this.

Reply:
No problem! Additional validation is no more.